### ISSUE TITLE
Key separator at the end of string not handled properly

### ISSUE THREAD

**Issue by dmgawel** [locize/gettext-converter#9]
## 🐛 Bug Report

Strings with separator at the end are not properly converted back from PO to i18next and JSON.

This is quite important when using `i18next` missing keys resolution with dynamic content (no named keys), as `.` is the common separator and also often appears at the end of string. In that case saving the missing string `t('This is a sample sentence.')` produces JSON like this:

```json
{  
  "This is a sample sentence": {
    "": "This is a sample sentence."
  }
}
```

## To Reproduce

I've created a draft PR with updated test fixtures, to cover this in tests: https://github.com/locize/gettext-converter/pull/10. Feel free to take it over.

In a nutshell, this i18next JSON:

```json
{  
  "Nested empty key": {
    "": "test"
  }
}
```

produces the following PO:

```po
msgid "Nested empty key##"
msgstr "test"
```

but converting it back gives:

```json
{  
  "Nested empty key##": "test"
}
```

## Expected behavior

Converting back the above example should output:

```json
{  
  "Nested empty key": {
    "": "test"
  }
}
```

## Your Environment

- *runtime version*: node v16
- *module version*: 1.2.3
- *os*: Mac

**Comment by adrai** [comment]
@perrin4869 what do you think? imo this looks like a strange/wrong i18next json:

```json
{  
  "Nested empty key": {
    "": "test"
  }
}
```

**Comment by adrai** [comment]
v1.2.4 should address this

**Comment by dmgawel** [comment]
@adrai Works great, thank you!
